backend/dal/Services/Admin/Concrete/AgencyService.cs

```diff
@@ -31,7 +31,7 @@ public class AgencyService : BaseService<Agency>, IAgencyService
         #endregion
 
         #region Methods
-        
+
```

Comment:
A lot of ugly updates to this service...  Essentially we need to either update the users affected by agency updates, or we need to return a list of users who will be affected by the agency update so that keycloak can be updated.